core/index/src/main/java/mil/nga/giat/geowave/core/index/IndexUtils.java

```diff
@@ -21,4 +21,42 @@ public static MultiDimensionalNumericData getFullBounds(
 		return new BasicNumericDataset(
 				boundsPerDimension);
 	}
+
+	public static final double getDimensionalBitsUsed(
+			final NumericIndexStrategy indexStrategy,
+			final double[] dataRangePerDimension ) {
+		double result = Long.MAX_VALUE;
+		if (dataRangePerDimension.length == 0) {
+			return 0;
+		}
+		final double cellRangePerDimension[] = indexStrategy.getHighestPrecisionIdRangePerDimension();
+		final double bitsPerDimension[] = getBitsPerDimension(
+				indexStrategy,
+				cellRangePerDimension);
+		for (int d = 0; d < dataRangePerDimension.length; d++) {
+			final double bitsUsed = bitsPerDimension[d] - log2(Math.ceil(dataRangePerDimension[d] / cellRangePerDimension[d]));
```

Comment:
The prior approach created a ratio bits required over bits available.   Minimizing the ratio was preference to use the least number of bits.  This, I believe, was aligned to the idea we discussed of looking at the number bits from the left..
This approach states that the result is bits used.  However, it seems to me this is bit unused: the total bits subtracting the bits required.  I would think that this should be maximized.